apps/vaporgui/FlowEventRouter.cpp

```diff
@@ -62,6 +69,7 @@ FlowEventRouter::FlowEventRouter( QWidget *parent, ControlExec *ce)
 FlowEventRouter::~FlowEventRouter(){
 	if (_variables)  delete _variables;
 	if (_geometry)   delete _geometry;
+	if (_seeding)    delete _seeding;
 	if (_appearance) delete _appearance;
 	if (_annotation) delete _annotation;
 }
```

Comment:
Who has the ownership of these widgets, (i.e., _variables, _geometry, _seeding) and who is responsible of cleaning them up?

Reply:
These are all subtabs (not TFWidgets, GeometryWidgets etc) and are all owned by the EventRouter.  SubTabs do not share a base class.  They just hold GUI components.

EventRouter -> SubTab -> TabWidgets

Reply:
You're correct.  I've removed the redundant deletions.